publication-event-handlers/src/main/java/no/unit/nva/publication/events/handlers/delete/ScopusEmitDeletionEventHandler.java

```diff
@@ -0,0 +1,98 @@
+package no.unit.nva.publication.events.handlers.delete;
+
+import static java.util.UUID.randomUUID;
+import static nva.commons.core.attempt.Try.attempt;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.events.S3Event;
+import java.net.URI;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import no.unit.nva.publication.events.bodies.ScopusDeletionEvent;
+import no.unit.nva.s3.S3Driver;
+import nva.commons.core.Environment;
+import nva.commons.core.JacocoGenerated;
+import nva.commons.core.paths.UnixPath;
+import nva.commons.core.paths.UriWrapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import software.amazon.awssdk.services.s3.S3Client;
+
+public class ScopusEmitDeletionEventHandler implements RequestHandler<S3Event, Void> {
+
+    public static final String S3_URI_TEMPLATE = "s3://%s/%s";
+    public static final String SCOPUS_IDENTIFIER_DELIMITER = "DELETE-";
+    public static final String EVENTS_BUCKET = new Environment().readEnv("EVENTS_BUCKET");
+    public static final String EMITTED_EVENT_LOG_MESSAGE = "Emitted Event:{}";
+    private static final int SINGLE_EXPECTED_FILE = 0;
+    private static final Logger logger = LoggerFactory.getLogger(ScopusEmitDeletionEventHandler.class);
+    private final S3Client s3Client;
+
+    @JacocoGenerated
+    public ScopusEmitDeletionEventHandler() {
+        this(S3Driver.defaultS3Client().build());
+    }
+
+    public ScopusEmitDeletionEventHandler(S3Client s3Client) {
+        super();
+        this.s3Client = s3Client;
+    }
+
+    @Override
+    public Void handleRequest(S3Event input, Context context) {
+        Optional.ofNullable(readFile(input))
+            .map(this::extractIdentifiers)
+            .stream().flatMap(List::stream)
+            .forEach(this::emitDeletionEvent);
+
+        return null;
+    }
+
+    private static List<String> splitOnNewLine(String string) {
+        return Arrays.asList(string.split("\n"));
+    }
+
+    private static boolean isEmptyLine(String string) {
+        return !string.isBlank() || !string.isEmpty();
+    }
+
+    private void emitDeletionEvent(String item) {
+        var event = new ScopusDeletionEvent(ScopusDeletionEvent.EVENT_TOPIC, item);
+        var s3Driver = new S3Driver(s3Client, EVENTS_BUCKET);
+        var eventPath = UnixPath.of(randomUUID().toString());
+        attempt(() -> s3Driver.insertFile(eventPath, event.toJsonString()));
```

Comment:
Ehh, I think you are using the wrong aws service. You should be using BatchEventEmitter, not putting them inside s3.